releasenotes/notes/adding-TextEmbedder-Protocol-948b31b005d798b9.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    We now have a Protocol for TextEmbedder. One impact is that now we can build custom components or SuperComponents that are not tied to a specific text embedder.
```

Comment:
I'd say it was possible before with something like `Union["EmbedderA", "EmbedderB"]`. I suggest we change the second sentence to. The protocol makes it easier to create custom components or SuperComponents that expect any TextEmbedder as init parameter.

Reply:
make sense yes, agree 👍🏽